src/engine/basics/Settings.hxx

```diff
@@ -166,6 +166,15 @@ struct SettingsData
 
   /// Write errors to a log file
   bool writeErrorLogFile;
+
+  // ==================================
+  // Command line options
+  // ==================================
+
+  /// Sets a different video driver
+  std::string videoDriver = "Default";
```

Comment:
Will video driver handling be added later in a different PR? I guess I'm also just wondering why this parameter is necessary, like I don't know how many people use different drivers for different programs. But then again, I don't normally run Cytopia from the command line.